lib/web_ui/lib/src/engine/platform_dispatcher.dart

```diff
@@ -135,6 +135,13 @@ class EnginePlatformDispatcher extends ui.PlatformDispatcher {
     viewData[view.viewId] = view;
   }
 
+  /// Removes [view] from the platform dispatcher's registry of [views].
+  ///
+  /// Nothing happens if the view is not already registered.
+  void unregisterView(EngineFlutterView view) {
+    viewData.remove(view.viewId);
```

Comment:
Do we need to call `dispose` on the `view`?

Reply:
There's no `dispose` on `view`. But if there was a `FlutterView.dispose` method, then here's how it would look like:
```dart
class EngineFlutterView implements ui.FlutterView {
  EngineFlutterView(...) {
    // This already exists.
    platformDispatcher.registerView(this);
  }

  void dispose() {
    // This doesn't exists yet, but this is what `dispose` would look like if we ever add one (I think we will).
    platformDispatcher.unregisterView(this);
  }
}
```

Reply:
I guess what I'm saying is: if we ever have a `FlutterView.dispose`, then `FlutterView.dispose` should call `PlatformDispatcher.unregisterView` and not vice versa.

Reply:
Ha-ha. Gotcha.